README.md

```diff
@@ -79,12 +81,10 @@ I've included some plotting recipes for convenience using [`RecipesBase`][3].
 ```julia
 using StatPlots
 
-abund = AbundanceTable(
-    rand(100, 10), ["sample_$x" for x in 1:10],
-    ["feature_$x" for x in 1:100])
+abund = abundancetable(rand(100, 10))
```

Comment:
The standard names for samples and features seemed so relevant I added them as a default to the constructor